readme.md

```diff
@@ -56,20 +68,17 @@
     - C++ /CLI support for v143 build Tools (Latest)  (for OMOD and FOMOD installers)
     - Windows 11 SDK (get latest)
     - C++ Build Tools core features
-  - Optional:
-    - Git for Windows (if you don't have git already installed)
```

Comment:
If this isn't installed as a VS Individual Component, then there's no advantage to using an x64 Native Tools Command Prompt for VS 2022 over regular CMD/PowerShell, so I wouldn't remove this line. In fact, adding another one for *C++ CMake tools for Windows* would probably be a good idea.

Reply:
Updated